src/content/getting-started/best-practices/index.md

```diff
@@ -111,7 +111,7 @@ The downside to the multi-tenant single cluster approach is that it, as mentione
 ## Further reading
 
 - [Creating clusters with gsctl]({{< relref "/use-the-api/gsctl/create-cluster" >}})
-- [Cluster Size and Autoscaling]({{< relref "/getting-started/cluster-size-autoscaling" >}})
+- [Cluster Size and Autoscaling]({{< relref "/getting-started/operations/autoscaling/cluster-size" >}})
```

Comment:
Should it be in "getting-started" or in "cost optimization" (https://docs.giantswarm.io/advanced/cost-optimization/) section? 
There's already some notes about Goldilocks and VPA here: https://docs.giantswarm.io/advanced/cost-optimization/tools/

Reply:
I think how to use it should be in getting-started as it's part of our defaults tbh. Also, the nodes regarding Goldilocks and VPA could lead to this new doc eventually